app/views/foreman_ansible/job_templates/download_and_execute_script.erb

```diff
@@ -0,0 +1,24 @@
+<%#
+name: Download and execute a script
+job_category: Ansible Commands
+description_format: Download script from %{url} and execute it
+snippet: false
+template_inputs:
+- name: url
+  required: true
+  input_type: user
+  description: "URL of the script to download e.g: http://example.com/unattended/anonymous/my_script"
+  advanced: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+---
+- hosts: all
+  tasks:
+    - shell:
```

Comment:
Would it be better to use the [url](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/url_lookup.html) module to retrieve the content and then execute the retrieved content?